# Домашнее задание

## Теория вероятностей

### **Задача 1**. (11 баллов)

В известном магазине техники представлено очень много экземпляров различной бытовой техники двух видов: с возможностью подключения к системе "Умный дом" и без такой возможности. Причем, известно, что эти два вида техники представлены в пропорции 1 : 3 соответственно. Случайным образом выбрали 7 экземпляров техники для розыгрыша среди участниц конкурса "Мисс Рукоделье - 2024".  Какова вероятность, что среди них не более двух экземпляров с возможностью подключения к системе "Умный дом"?

Подробно распишите решение и обоснуйте выбор распределения для моделирования сюжета задачи. Ответ округлите до трёх знаков.


**Решение**

Так как в задаче описываются события имеющие ровно два исхода, успех (не более двух экземпляров в 7), при этом вероятность каждого успеха и неуспеха постоянна, а выборки большие и независимые, значит мы имеем дело с биноминальным распределением.

![подробное решение](dz_2_1.jpg)

### Ответ:
Вероятность, что среди 7 случайно выбранных экземпляров будет не более двух экземпляров с возможностью подключения к системе "Умный дом" примерно равна $75.5$% - $75.6$%.

### **Задача 2** (30 баллов)

Дан закон распределения количества публикаций в которой группе телеграм-каналов:

$$
\begin{array}{|c|c|c|c|c|c|c|}
\hline X \text { (число публикаций) } & 0 & 1 & 2 & 3 & 4 & 5 \\
\hline \mathrm{P} \text { (вероятность) } & ? & 0,3 & 0,3 & 0,2 & 0,1 & 0,1 \\
\hline
\end{array}
$$

Найдите пропущенную вероятность для X = 0, математическое ожидание и дисперсию. Решите задачу "вручную" и с помощью python.

#### Решение вручную:

![Задача 2 решение](dz_2_2.jpg)

#### Решение с помощью Python:

In [2]:
import numpy as np

In [5]:
#Данные
x = np.array([0, 1, 2, 3, 4, 5,])
p = np.array([0, 0.3, 0.3, 0.2, 0.1, 0.1,])
#проверка суммы вероятностей, P=1 - по определению
p.sum()

1.0

In [20]:
#математическое ожидание
ex = np.sum(x * p)
ex

2.4

In [21]:
#дисперсия
ex2 = np.sum(x**2 * p)
ex2 - ex**2

1.6400000000000006

### Ответ

$P(X=0)=0,
E[X]=2.4,
Var(X)=1.64$

### **Задача 3** (25 баллов)


Компания приобрела 10 000 новых ноутбуков. Вероятность поломки одного ноутбука в течение месяца равна 0,0002. Предполагается, что поломки отдельных ноутбуков происходят независимо друг от друга.

#### **Вопрос 1**. Найдите вероятность того, что за месяц сломается ровно 1 ноутбук.
Решение объясните. (5 баллов)


##### Решение:

![Задача 3а](dz_2_3_a.jpg)

##### Объяснение:

Выбираем распределение Пуассона, так как имеем дискретное распределение вероятностей числа событий, происходящих в данный период времени, с учётом того, что мы можем вычислить среднее количество раз, когда событие произойдёт за этот период времени.

Чтобы рассчитать вероятность того, что за период времени произойдёт $k$ событий, можно воспользоваться следующей формулой:

$P(X = k) = \frac{\lambda^k e^{- \lambda}}{k!}$


#### **Вопрос 2а**. 
Компания планирует ввести программу расширенной гарантии, по которой она будет бесплатно ремонтировать или заменять ноутбуки, сломавшиеся в течение месяца. Если общее количество поломок за месяц превысит 5 ноутбуков, компания понесет значительные дополнительные расходы.

а) Найдите вероятность того, что за месяц сломается более 5 ноутбуков.Решение объясните (10 баллов)

##### Решение

Так как нам надо найти вероятность того, что за месяц сломается более 5 ноутуков, то это равно единица минус сумма вероятностей того, что за месяц сломается 0 ноутбуков, 1 ноутбук, 2, 3, 4, 5 ноутбуков. Подробное решение на скрине выше.

#### **Вопрос 2б**. 

б) Если стоимость ремонта или замены одного ноутбука составляет 500 долларов, вычислите математическое ожидание общих расходов компании на поломки за один месяц. (10 баллов)

##### Решение:

![Задача 3 2б](dz_2_3_b.jpg)

### Ответ:

$P(X=1)≈0,271$

а) $P(X>5)≈0,017$

б) Ожидаемые расходы = $1000$ долларов.

